test/11-api-binder.spec.ts

```diff
@@ -65,6 +60,14 @@ const mockProvisioningOpts = {
 describe('ApiBinder', () => {
 	let server: Server;
 
+	beforeEach(() => {
+		stub(eventTracker, 'track');
+	});
+	afterEach(() => {
+		// @ts-expect-error Restoring a non-stub type function
```

Comment:
This is more of a `ts-ignore` because if the typings were magical enough then this would be valid whereas `@ts-expect-error` is more if you'd always expect an error, eg `const s: string = new Date()` - for reference if you did
```ts
// @ts-expect-error
const s: string = 'valid string'
```
then it would actually error as it expects it to fail and the fact that it doesn't is treated as an issue